.eslintrc.yml

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Can you rename this to file to .eslintrc instead
Reason being that the gulp task runner for linting will be looking for a .eslintrc file in the root directory to apply rules